### DISCUSSION TITLE
Avatar floating

### DISCUSSION THREAD

**Discussion by prismalx** [mozilla/hubs#4006] [Spoke Questions] [answered]
Hello everyone. I have an issue with my scene created in spoke which i already open as a topic in support.mozilla.org. I know i shouldnt open a topic here while the other is pending but in the other channel im kind of stuck and the answers are always coming up with a big delay of several days. im kind of getting under time pressure now so i thought to ask here for help. 

we created a room (last link https://hubs.mozilla.com/ioQxDs9/infamous-glamorous-party) on spoke which is a 3d modell of our art association in lisbon. when the avatar enters the room he is always floating above the ground. we tried already

a) to define under "spawn point" the y position ( 0 and even minus values) and bring it closer to the floor
b) to bring the y position of of model (tall.glb) higher (1 or even higher values)
c) raise up the spawn point up as suggested in the other help threat

none of them solved my problem.

when he floats it feels unrealistic to "walk" through the space. as we want to host online exhibitions of our art association in this room we are trying to reach a more realistic approach.

here you can read what answers i got so far from the help support - non of them fixed my problem. https://support.mozilla.org/en-US/questions/1327865

thanks a lot for your help!!

**Comment by johnshaughnessy** [comment, marked as answer]
Closing this as there's already a thread with more detail in the support site https://support.mozilla.org/en-US/questions/1327865
